gym/dev_wrappers/lambda_action.py

```diff
@@ -0,0 +1,174 @@
+"""Lambda action wrappers that uses jumpy for compatibility with jax (i.e. brax) and numpy environments."""
+
+from typing import Any, Callable
+from typing import Tuple as TypingTuple
```

Comment:
What is the point of this? There isn't any other `Tuple` here. If there was (e.g. from spaces), I think it'd be much clearer to just directly use `typing.Tuple`

Reply:
Yes I've used it quite a lot for collision with Tuple and Dict spaces, typing.Tuple seems better :+1: 
I'm going to update it also in other part of the code